docs/content.zh/docs/learn-flink/etl.md

```diff
@@ -156,56 +154,50 @@ keyBy(ride -> GeoUtils.mapToGridCell(ride.startLon, ride.startLat))
 
 ### Keyed Stream 的聚合
 
-以下代码为每个行程结束事件创建了一个新的包含 `startCell` 和时长（分钟）的元组流：
+以下代码为每个行程结束事件创建了一个新的包含 `startCell` 和距离的元组流：
 
 ```java
-import org.joda.time.Interval;
-
-DataStream<Tuple2<Integer, Minutes>> minutesByStartCell = enrichedNYCRides
-    .flatMap(new FlatMapFunction<EnrichedRide, Tuple2<Integer, Minutes>>() {
+DataStream<Tuple2<Integer, Double>> minutesByStartCell = enrichedNYCRides
+    .flatMap(new FlatMapFunction<EnrichedRide, Tuple2<Integer, Double>>() {
 
         @Override
         public void flatMap(EnrichedRide ride,
-                            Collector<Tuple2<Integer, Minutes>> out) throws Exception {
-            if (!ride.isStart) {
-                Interval rideInterval = new Interval(ride.startTime, ride.endTime);
-                Minutes duration = rideInterval.toDuration().toStandardMinutes();
-                out.collect(new Tuple2<>(ride.startCell, duration));
+                            Collector<Tuple2<Integer, Double>> out) throws Exception {
+            if (ride.isStart) {
+                double distance = ride.getEuclideanDistance(ride.endLon, ride.endLat);
```

Comment:
and these args should be ride.startLon and startLat

Reply:
As I mentioned above, this is because I choose the start ride event. Thus, `endLon` and `endLat` are used here.